tests/parser/tests/cases/should_fail/integer_literal_too_large/case.hash

```diff
@@ -0,0 +1 @@
+let k = 23384912398412397432849123413027948091234809213480923409823231498;
```

Comment:
What is the maximum integer literal supported?

Reply:
Also, what about negative literals?

Reply:
The maximum integer supported is a `u64`, however, it maybe problematic with signed literals as then it becomes 2^6 3

Reply:
We could get around this by not converting to negative straight away and using the `neg` trait?

Reply:
hmm, true. Maybe that is indeed a good idea. However, negative integer literals won't be able to be represented up to -(2^64-1) anyway in the language (it will be -(2^63-1) again).